src/Components/Facility/ConsultationABGTab.tsx

```diff
@@ -0,0 +1,23 @@
+import { ConsultationTabProps } from "../../Common/constants";
+import PageTitle from "../Common/PageTitle";
+import { ABGPlots } from "./Consultations/ABGPlots";
+export default function ConsultationABGTab({
+  facilityId,
+  patientId,
+  consultationId,
+}: ConsultationTabProps) {
+  return (
+    <div>
+      <PageTitle
+        title="ABG Analysis Plot"
+        hideBack={true}
+        breadcrumbs={false}
+      />
```

Comment:
It would be nice if you could wrap the actual children (like ABGPlots...) in `Page` component instead without having a parent div. This is to ensure we have consistent padding across all page's. `PageTitle` is a legacy component.

The above changes can be done for all other tabs as well.

Reply:
implemented